### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by tiftran**

### CODE REVIEW

File: app/requirements/default.txt [hunk 1]
```diff
@@ -167,5 +166,5 @@ jsonschema==3.2.0 \
     --hash=sha256:4e5b3cf8216f577bee9ce139cbe72eca3ea4f292ec60928ff24758ce626cd163 \
     --hash=sha256:c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
 mozilla-nimbus-shared==0.0.9 \
-    --hash=sha256:7f2c8c24ea66fefac42e94e1cae8c3232cfdf3b9a8daf8b8fb738037218d82aa \
-    --hash=sha256:98542a8d4e0b75ee2394eb9c25775f63d5ef31fed0f8a38ce5bb2835e6e64b1b
+    --hash=sha256:98542a8d4e0b75ee2394eb9c25775f63d5ef31fed0f8a38ce5bb2835e6e64b1b \
+    --hash=sha256:7f2c8c24ea66fefac42e94e1cae8c3232cfdf3b9a8daf8b8fb738037218d82aa
```
Comment (pdehaan) [inline app/requirements/default.txt:170]: Curious why the SHAs would change if the version didnt.

Comment (tiftran) [reply app/requirements/default.txt:170]: oooh, bad merge conflict resolution me thinks, will fix

Comment (jaredlockhart) [reply app/requirements/default.txt:170]: Oh wait why why was that missed....

Comment (pdehaan) [reply app/requirements/default.txt:170]: Or... wait... maybe the values are the same, just reversed now.

Comment (tiftran) [reply app/requirements/default.txt:170]: wait.... seems like you're right..., they're reversed...

**Pull request merged**